lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -72,8 +72,10 @@ class DetailsRenderer {
    * @return {!Element}
    */
   _renderURL(text) {
-    const element = this._renderText(text);
+    const parsedURL = Util.parseURL(text.text);
```

Comment:
we're going to have to try to be backwards compatible here now that we've released

maybe we can make parseURL do this safely, accept `{string|undefined}` and return null if the URL was not parsable for some reason? then in here we can check if we were able to parse and either set the tooltip with the full text or fallback to the original behavior

on the plus side this should address both the linting and the test failures :)